client/blocks/reader-feed-item/index.tsx

```diff
@@ -171,12 +214,11 @@ export default function ReaderFeedItem( props: ReaderFeedItemProps ): JSX.Elemen
 	const SubscribeButton = (): JSX.Element => (
 		<Button
 			variant="primary"
-			disabled={ subscribeDisabled }
```

Comment:
we may want to disable in the same cases as isBusy. It looks like busy doesn't disable and you can multi-click to get errors while the actions are pending.

Reply:
Nice Catch. Let me fix it.

Reply:
Fixed [6bbbe0f](https://github.com/Automattic/wp-calypso/pull/101042/commits/6bbbe0fd221e43e3a79609a72ceb7fb7626b61c1)